scripts/hal_alignment/hal_gene_liftover.py

```diff
@@ -49,17 +49,42 @@ class SimpleRegion(NamedTuple):
     strand: str
 
 
+def load_chr_sizes(hal_file: Union[Path, str], genome_name: str) -> Dict[str, int]:
```

Comment:
I forgot we have also defined in our library `from ensembl.compara.filesys import PathLike`, so you may want to use that one instead:
```suggestion
def load_chr_sizes(hal_file: PathLike, genome_name: str) -> Dict[str, int]:
```
(and then you can remove `from pathlib import Path`)

Reply:
You make a compelling case for using `ensembl.compara.filesys.PathLike`, especially given the amount of path-handling that this script is expected to do. But I would make this change in a dedicated pull request, as it would necessitate some additional adjustments.